go.mod

```diff
@@ -177,3 +173,5 @@ replace (
 	// command line flags and conflicting with command line flags added by libbeat.
 	github.com/golang/glog => ./internal/glog
 )
+
+replace github.com/dgraph-io/ristretto v0.1.2-0.20240116140435-c67e07994f91 => github.com/dgraph-io/ristretto v0.2.0
```

Comment:
can we add a comment above on why we are doing thing?

Reply:
how is this even working as intended?
the API might have changed between 0.1 and 0.2 🤔 

is there another dependency that requires 0.1? otherwise I don't see why Go modules MVS sholdn't figure out autonomously to use 0.2